_posts/2022-11-21-trino-summit-2022-apple-recap.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Trino Summit 2022 recap: Trino at Apple"
```

Comment:
Do we need such a long title .. how about just "Trino at Apple"

Reply:
Well I guess as long as Apple never titles their session the same lol

Reply:
Yeah .. I know.. hence I am not sure.. but at least the date and content will always be different..